coralproject/talk
# [CORL-554] Un-feature a comment when it is rejected

Also decrements the featured count for the story if rejected using the
in-stream moderation.

## How to test

- Sign in as a regular user
- Create a comment
- Sign in as moderator
- Go to stream
- Feature the comment
- Reject the comment
- See how comment is rejected (disappears from stream)
- See how featured count goes down by 1
